scripts/component_docs_generator/helpers/file_downloader.go

```diff
@@ -0,0 +1,117 @@
+package helpers
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+)
+
+func GetIntegrationDocsCSVFile(csvURL string) {
```

Comment:
Meshkit has similar function available to get file, try using that.

Reply:
@MUzairS15, what's the goal here - to have three separate scripts?

No, right? Instead each of them need to be rewritten into MeshKit and used by Meshery Server and mesheryctl. Given this, shall we forego the effort to try to make this one robust and instead accept the bad with the good in order to use this for the short-term?

@aboobakersiddiqr63 do you have sample usage of this temp script? Will you create a readme.md and more urgently, will you push all of the generated integration files to the Meshery Docs site, so that users can reference them?

Reply:
@leecalcote will do it today

Reply:
@MUzairS15 ?